test/units/testsuite-73.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+set -eux
+set -o pipefail
+
+setup_cron() {
+    # Setup test user and cron
+    useradd test
+    echo "test" | passwd --stdin test
+    crond -s -n &
+    # Install crontab for the test user that runs sleep every minute. But let's sleep for
+    # 65 seconds to make sure there is overlap between two consecutive runs, i.e. we have
+    # always a cron session running.
+    crontab -u test - <<EOF
+RANDOM_DELAY=0
+* * * * * /bin/sleep 65
+EOF
+    # Let's a bit more than one interval to make sure that cron session is started already
+    sleep 70
```

Comment:
To reduce runtime, how about to run a loop to check at least one session is started?

Reply:
Ok